dipper/curie_map.yaml

```diff
@@ -1,3 +1,7 @@
+# Skolemize Blank Nodes
+# overwrite iri fragment with '' for unresovable bnodes
+'_':    'https://monarchinitiative.org/.well-known/genid/'
```

Comment:
This really needs to be well documented. In RDF the concept of a blank node is unambiguous. Once you skolemize into a URI, it's not anonymous, period.

I see the use case for being able to tag used-to-be-blank nodes, but I don't think we should confuse terminology. We should call these pseudo-anonymous or something. And we need something on the main scigraph docs about them (or if this is something purely localized to monarch convention, we still need this documented somewhere).

I also think encoding a secret convention between scigraph and dipper about URI syntax is a bit fragile. Probably best to keep right now, but really this should be more explicit. E.g. if dipper wants to mark a URI in some way, then it should mark is explicitly (with a triple), not by convention